epub34/a11y/index.html

```diff
@@ -745,14 +733,13 @@ <h6>Page list</h6>
 								<dt id="sec-page-list-conf">Meeting this Objective</dt>
 								<dd>
 									<p>An EPUB publication MUST include a page list.</p>
-									<p><a href="https://www.w3.org/TR/epub/#dfn-epub-creator">EPUB creators</a> SHOULD
-										include links to all pages of content reproduced from the source (i.e., they do
-										not have to provide links for blank pages or content not reproduced in the
-										digital edition).</p>
-									<p>EPUB creators MUST include links to all <a href="#sec-page-breaks-obj">page break
-											markers</a> in the content.</p>
-									<p>EPUB creators should include links for all pages in the source whether they are
-										reproduced or not, but this is not a requirement.</p>
+									<p>The page list SHOULD include links to all pages of content reproduced from the
+										source (i.e., it does not have to include links for blank pages or content not
+										reproduced in the digital edition).</p>
+									<p>The page list MUST include links to all the <a href="#sec-page-breaks-obj">page
+											break markers</a> in the content.</p>
+									<p>The page list should include links for all pages in the source, whether they are
+										reproduced or not, but this is not a conformance requirement.</p>
```

Comment:
```suggestion
									<p>The page list MUST include links to all the <a href="#sec-page-breaks-obj">page
											break markers</a> in the content.</p>
									<p>The page list should include links to all pages in the source, whether they are
										reproduced or not, but this is not a conformance requirement.</p>
```

Just being consistent...